### ISSUE TITLE
Need a network config option for TouchServer

### ISSUE THREAD

**Issue by pwn0r** [Akshayaap/TouchServer#1]
I spend a bit of time researching why it does not work in my setups, two different computers on Windows10 and three mobiles running various versions of LOS. 

then I looked in the server code (im not good with java so no point in looking in app code). It appears that your app finds the server as the server repeatedly broadcasts on ECHO port (which is hardcoded now as 5560 I believe). 

The problem is that the server sends on the wrong IP in my case. I have vmware/virtualbox installed on the machines, so for some reason the server picks (in both cases) a virtual network interface instead of a real ethernet connection (which is on the same network as wifi). 

Basically I guess we would need a config file where networking options can be manually specified. Network adapter selection, IP address and subnet etc.  

As a side note the program assumes the subnet is 255.255.255 in effect (it only adjusts the last byte of IP4). While this is the most common variant with the routers, there is no guarantee for any particular network to be this way.

**Comment by Akshayaap** [comment]
We are extremely sorry about that. we will fix it very soon.
We are also looking for someone who can contribute in this project. if you are interested, please let us know.

**Comment by TheShanMan** [comment]
I think I'm running into this too. The app never connects even if I disable windows firewall.
